RTLIB.md

```diff
@@ -0,0 +1,54 @@
+JULEP RTLIB
+===========
+
+-	**Title:** A runtime-library for Julia
+-	**Authors:** Valentin Churavy \<v.churavy@gmail.com\>
+-	**Created:** November 11, 2016
+-	**Status:** work in progress
+
+Problem
+-------
+
+-	How do we support `Float16` and `Float128` in a portable and performant manner?
+-	Can we implement Julia intrinsics in Julia.
+-	Provide a definition for necessary and optional intrinsics, that can be shared across compilers. Especially important for Julia compilers that are implemented in Julia (e.g. the Julia GPU compiler).
+
+Introduction
+------------
+
+Currently there are two implementation of intrinsics supported in Julia. One of the implementations is in `runtime_intrinsics.c` and the other one is implemented in `intrinsics.cpp` on top of [LLVM intrinsics](http://llvm.org/docs/LangRef.html#id1190) and [LLVM instructions](http://llvm.org/docs/LangRef.html#instruction-reference).
+
+The first implementation specifies the semantics and behaviour of the intrinsics and is used as a fallback. The LLVM based implementation is best understood as a pure performance optimization.
```

Comment:
This mixes British and American English spellings. In particular, "behaviour" is BE and "optimization" is AE. For consistency they should probably be either "behavior" and "optimization" or "behaviour" and "optimisation."